src/codacy-coffeescript.coffee

```diff
@@ -5,7 +5,9 @@ defaultLintConfig = require './default'
 lintConfig = {}
 
 try
-    userConfig = require '/src/.codacy'
```

Comment:
Why did we change this from the require to the `readFileSync`?

Reply:
require expects a file with the .json extension and .codacyrc does not have an extension. Basically using require with codacyrc behaves like the file was not found

Reply:
👍